src/style.scss

```diff
@@ -13,16 +13,16 @@
 @import "./node_modules/@alaskaairux/webcorestylesheets/dist/auroElement/auroElement";
 
 // container styles - defines the look of the outer custom component
-:host([error]),
-:host([warning]),
-:host([information]),
-:host([success]) {
+:host([type="error"]),
```

Comment:
When you migrate these will need to be single quotes. Might as well get ahead of the game with this change.